eng/common/instructions/azsdk-tools/local-sdk-workflow.instructions.md

```diff
@@ -5,6 +5,12 @@ description: "Guide the user to generate and build SDKs locally for a TypeSpec b
 # Goal
 
 Help the user generate and build SDKs locally from TypeSpec API specifications using the `azure-sdk-mcp` tools.
+High level steps involved:
+1. Generate SDK locally
+2. Build / Compile SDK locally
+3. Run package checks
+4. Run package tests
+5. Update change log, metadata and version
```

Comment:
```suggestion
High level steps involved:
1. Generate SDK locally
2. Build / Compile SDK locally
3. Run package tests
4. Update change log, metadata and version
5. Run package checks
```
You might need to switch these because come of the check, validate the updates to metadata and docs so they need to be done after those are updated.

Reply:
Package checks need to be run again before creating the PR.